CHANGELOG.md

```diff
@@ -23,6 +23,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Plots of objects defined by shape intersection logic will no longer display thin line artifacts.
 - Fixed incorrect gradient computation in PyTorch plugin (`to_torch`) for functions returning multi-element arrays.
 
+### Changed
```

Comment:
can you double check this? looks like a lot of extra items slipped in, is it expected?

This seems to add a whole new ### Changed section. Please add your change to the existing one.